.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Code owners for automatic review requests
+
+# Default owners for everything
+*       @topgrade-rs/maintainers
+
+# Workflows and CI
+.github/workflows/*  @topgrade-rs/maintainers
+
+# Docs tooling configs
+.markdownlint.json    @topgrade-rs/maintainers
+dprint.json           @topgrade-rs/maintainers
+.lychee.toml          @topgrade-rs/maintainers
+.typos.toml           @topgrade-rs/maintainers
```

Comment:
What is the practical benefit of this? You're just assigning everything to the same maintainers group.